<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KLMO8CO" failures="1" tests="14" name="Regression" time="0.107" errors="0" timestamp="2024-06-21T16:44:49 IST">
  <testcase classname="test.day1" name="Demo1" time="0.005"/>
  <testcase classname="test.day1" name="Demo" time="0.006"/>
  <testcase classname="test.day1" name="Test1" time="0.002"/>
  <testcase classname="test.day2" name="SecondTest" time="0.004"/>
  <testcase classname="test.day3" name="WebLoginAPICarLoan" time="0.002"/>
  <testcase classname="test.day3" name="WebLoginCarLoan" time="0.002"/>
  <testcase classname="test.day4" name="plan" time="0.002"/>
  <testcase classname="test.day4" name="plan1" time="0.002"/>
  <testcase classname="test.day5" name="APILoginCarLogin" time="0.001"/>
  <testcase classname="test.day5" name="MobileLoginCarLogin" time="0.003"/>
  <testcase classname="test.day5" name="MobileLoginCarLogin" time="0.001"/>
  <testcase classname="test.day5" name="MobileLoginCarLogin" time="0.003"/>
  <testcase classname="test.day1" name="Demo2" time="0.002"/>
  <testcase classname="test.day5" name="WebLoginCarLogin" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day5.WebLoginCarLogin(day5.java:17)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- WebLoginCarLogin -->
</testsuite> <!-- Regression -->
